config.js.template

```diff
@@ -103,3 +109,9 @@ module.exports = {
   // Check the /health endpoint documentation for more info.
   considerHealthcheckWarnAsUnhealthy: true,
 };
+
+if (config.fireblocksApiSecretFile && fs.existsSync(config.fireblocksApiSecretFile)) {
+  config.fireblocksApiSecret = fs.readFileSync(config.fireblocksApiSecretFile, { encoding: 'utf8' }).trim();
+}
+
```

Comment:
Shouldn't we do this in the `/start` controller? It's better than having this code in the config file.

Reply:
This would remove some code from the docker config file as well

Reply:
Done